### PR TITLE
Adding Symfem

### PR DESCRIPTION
**Pull request by mscroggs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mscroggs** [comment]
@conda-forge/staged-recipes This PR is ready for review

**Comment by mscroggs** [comment]
@conda-forge-admin, please ping team

**Comment by mscroggs** [comment]
This PR is ready for review

File: recipes/symfem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - numpy
+    - python >=3.6
+    - pytest
```
Comment (dopplershift) [inline recipes/symfem/meta.yaml:24]: It doesn't look like `pytest` is actually needed to use the library?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Is @symfembot a bot account? If so, is there a reason to have that account able to merge PRs?

File: recipes/symfem/meta.yaml (hunk 1 above)
Comment (mscroggs) [reply recipes/symfem/meta.yaml]: It isn't. I've remove it now

**Comment by mscroggs** [comment]
> Is @symfembot a bot account? If so, is there a reason to have that account able to merge PRs?

It is. I added it so I could automate updating of this, but I can automate it opening a PR without needed to allow it to merge them

**Comment by dopplershift** [comment]
@mscroggs Just FYI, conda-forge has a bot that monitors PyPI and GitHub for new releases and will also issue PRs automatically to the feedstocks to update them, so you might not want to invest time in your own automation.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**